quipucords/api/insights_report/test_serializers.py

```diff
@@ -27,11 +27,12 @@ def report_entity():
     return ReportEntity.from_report_id(deployment_report.id)
 
 
+# pylint: disable=redefined-outer-name
```

Comment:
huh. this is odd. pylint-pytest plugin should be preventing this check in tests.

are you sure this is required?

Reply:
Lint wont pass neither on the pipeline nor locally if this is removed.

Reply:
what if you clean pylint cache (`rm ~/.cache/pylint`) and then run `flakeheaven lint` instead of `make lint`?

Reply:
It still complains about it. I thought it could be a problem in my local config but it also broke on the lint step in the gh pipeline.

Reply:
fwiw, seems to be ok without it on the mac.

```
abellotti $ git diff
diff --git a/quipucords/api/insights_report/test_serializers.py b/quipucords/api/insights_report/test_serializers.py
index 6b5e268a..66043f66 100644
--- a/quipucords/api/insights_report/test_serializers.py
+++ b/quipucords/api/insights_report/test_serializers.py
@@ -27,7 +27,6 @@ def report_entity():
     return ReportEntity.from_report_id(deployment_report.id)
 
 
-# pylint: disable=redefined-outer-name
 @pytest.mark.dbcompat
 def test_factset_serializer(db, report_entity):
     """Test FactsetSerializer."""
```

then followed with:

```
abellotti $ flakeheaven lint quipucords/api/insights_report/test_serializers.py
abellotti $
```

Not sure if this is package version related or not, I'm running with python 3.9.13
and the following pylint versions:

```
abellotti $ pip list | grep lint
pylint              2.15.6
pylint-django       2.5.3
pylint-plugin-utils 0.7
pylint-pytest       1.1.2
```

Reply:
I was able to perform an exorcism in my machine and got it working.